src/Template.php

```diff
@@ -96,6 +96,41 @@ public function file(): ?string
             return Kirby::instance()->extension($this->store(), $name);
         }
     }
+   
+    public function bladeFile(): ?string
+    {
+        if ($this->hasDefaultType() === true) {
+            try {
+                // Try the default template in the default template directory.
+                return F::realpath($this->getFilename(), $this->templatesPath);
+            } catch (Exception $e) {
+                //
+            }
+
+            // Look for the default template provided by an extension.
+            $path = Kirby::instance()->extension($this->store(), $this->name());
+
+            if ($path !== null) {
+                return $path;
+            }
+        }
+
+        // disallow blade extension for content representation, for ex: /blog.blade
+        if ($this->type() === 'blade') {
+            return null;
+        } else {
+            $name = $this->name() . "." . $this->type();
+        }
+
+        try {
+            // Try the template with type extension in the default template directory.
+            return F::realpath($this->getFilename($name), $this->templatesPath);
+        } catch (Exception $e) {
+            // Look for the template with type extension provided by an extension.
+            // This might be null if the template does not exist.
+            return Kirby::instance()->extension($this->store(), $name);
+        }
+    }
```

Comment:
This section seems to be a duplication of the method `file()`. Is there a way to merge this both methods?